app/routes/photos/components/GalleryPictureModal.tsx

```diff
@@ -376,22 +399,31 @@ const GalleryPictureModal = () => {
             </span>
           )}
 
-          <Flex justifyContent="center" gap="var(--spacing-lg)">
-            {hasPrevious && (
-              <Icon
-                onPress={previousGalleryPicture}
-                name="arrow-back-outline"
-                size={40}
-              />
-            )}
-            {hasNext && (
+          <Flex
+            justifyContent="center"
+            alignItems="center"
+            gap="var(--spacing-lg)"
+          >
+            <Icon
+              onPress={previousGalleryPicture}
+              name="arrow-back-outline"
+              size={40}
+              disabled={isFirstImage}
+            />
+            {isLastLoadedImage && !isLastImage ? (
+              <LoadingIndicator loading className={styles.loadingIndicator} />
+            ) : (
```

Comment:
Skeleton component? 😢

Reply:
This is on the next button. The current image is still displayed as normal:)
Looks like this while loading: 
![Screenshot 2024-11-05 at 17 15 17](https://github.com/user-attachments/assets/011ba5d1-1296-4ad3-8ddd-a76cae953261)